src/scripts/visualizations/legend.js

```diff
@@ -5,6 +5,7 @@
             vAlign: 'middle',
             hAlign: 'right',
             direction: 'vertical',
+            enabled: true,
```

Comment:
is there a reason to change the default? If not i'd recommend leaving the defaults as they were -- otherwise we'd need to go through all the published examples in the gallery and make sure they don't look weird.